### DISCUSSION TITLE
Notifiche anagrafiche con verifica lato ANPR

### DISCUSSION THREAD

**Discussion by Abstractconstruct** [italia/ansc#862] [Supporto servizi cooperativi] [answered]
Salve,

Volevamo chiedere per quanto riguarda la suddetta attività in cosa consisteva di preciso. Non ci è chiaro quali servizi cooperativi è associata.

Saluti,
Rimaniamo in attesa di un Vostro riscontro.

**Comment by Abstractconstruct** [comment]
Buongiorno,
ci sono aggiornamenti?

**Comment by Abstractconstruct** [comment]
Avendo ricevuto una risposta su una discussione precedente da @gbuemi .
Chiedo scusa in anticipo per il mention se inopportuno, essendo trascorsi circa due settimana dall'apertura della discussione.

**Reply by amusarra** [reply]
Salve @Abstractconstruct 
Potrebbe essere più chiaro? 
Dal solo oggetto della discussione non è chiaro il quesito. Sarebbe opportuno descrivere lo scenario.

Grazie.
Supporto ANSC.

**Reply by Abstractconstruct** [reply]
Salve @amusarra, 
nell'elenco delle attività necessarie (o servizi) per l'adozione controllata c'è questa attività che ho riportato lettera per lettera. Per il contesto abbiamo richiesto ad ANSC un excel con l'elenco delle attività con i casi d'uso da implementare e questa dicitura **Notifiche anagrafiche con verifica lato ANPR** fa parte dei servizi da realizzare.

Appunto dalla sola descrizione non ci è chiaro in cosa consiste e quindi come implementare l'attività.

**Reply by amusarra** [reply, marked as answer]
Salve @Abstractconstruct 
Grazie per aver esplicitato il contesto. 
L'attività **Notifiche anagrafiche con verifica lato ANPR** fa riferimento alla funzionalità [Notifiche predisposizione eventi digitali stato civile (ANSC)](https://github.com/italia/anpr/issues/3883) dove può trovare tutti i dettagli e qui https://www.anagrafenazionale.interno.it/area-tecnica/documentazione-tecnica/ può trovare tutta la documentazione tecnica per l'integrazione con i servizi di ANPR.

Cordiali Saluti.
Supporto ANSC.
